python/triton/language/core.py

```diff
@@ -921,7 +921,7 @@ def broadcast(input, other, _builder=None):
     :param other: The second input tensor.
     :type other: Block
     """
-    return semantic.broadcast_impl_value(input, other, _builder)
```

Comment:
This is a user-facing change, not sure we want to do this?  (The semantics of tt.broadcast in the IR do not have to match the semantics of tl.broadcast in Python.)